src/components/Budget/BudgetView/BudgetInformation.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Col,
+  Row,
+  KeyValue,
+} from '@folio/stripes/components';
+import {
+  AmountWithCurrencyField,
+  FolioFormattedDate,
+} from '@folio/stripes-acq-components';
+
+const BudgetInformation = ({ budget, fiscalStart, fiscalEnd }) => (
```

Comment:
it would be nice to pass primitives instead of object

Reply:
Ok